src/developer_config.yaml

```diff
@@ -22,22 +22,26 @@ runlog_writer:
     display: False # Display the runlog in the terminal
     log_path: "data/log"
 hdfs_paths:
-  root: None
-  logs_foldername: "/ons/rdbe_dev/logs/run_logs"
-  snapshot_path: "/ons/rdbe_dev/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
+  root: "/ons/rdbe_dev/"
```

Comment:
this needs to be updated to:
`  logs_foldername: "/ons/rdbe_dev/logs/run_logs"`

Reply:
Thanks for the fix